### PR TITLE
Toolbar: remove classes from the correct page when destroying fixedtoolbars

### PR DESCRIPTION
**Pull request by solymosi**
Should resolve issue #7995.

### CODE REVIEW

**Comment by solymosi** [comment]
Oops, I signed the CLA as "Máté Solymosi", whereas I use "Mate Solymosi" for committing. Signed it again using "Mate Solymosi".

**Comment by mdgross** [comment]
Thank you for this. I have tested the fix and it works.

**Comment by arschmitz** [comment]
@solymosi Thank you for looking into this. Unfortunately your fix while it fixes the problem your link to causes a regression when the toolbar is outside the page. The problem here is the logic needs to be different if the toolbar is internal vs external. @solymosi if you could modify your fix to check that and and apply it to the appropriate page that would work. We also would need some tests to guard against regressions .

**Comment by solymosi** [comment]
Thanks for the feedback. The regression should now be fixed. I simply copy-pasted the code that the `_setOptions` function uses to determine the page to operate on (first check the `page` property of the widget, and if that does not exist, only then use the currently active page).

File: js/widgets/fixedToolbar.js [hunk 1]
```diff
@@ -279,7 +279,7 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 
 		_destroy: function() {
 			var pageClasses, toolbarClasses, hasFixed, header, hasFullscreen,
-				page = this.pagecontainer.pagecontainer( "getActivePage" );
+				page = ( !!this.page )? this.page: ( $(".ui-page-active").length > 0 )? $(".ui-page-active"): $(".ui-page").eq(0);
```
Comment (arschmitz) [inline js/widgets/fixedToolbar.js]: I know you just copied this but can you add a space before  each `?` and `:` and inside all the parenthesized bodies

Comment (solymosi) [inline js/widgets/fixedToolbar.js]: Done.

**Comment by arschmitz** [comment]
Same thing with the rest of the js files as the one I commented lots of missing spaces :-) could you just ad those in?

**Comment by arschmitz** [comment]
Oh wow i didn't mean the whole files just the parts you were adding but thank you!

**Comment by arschmitz** [comment]
@solymosi This look good now thank you i will land this shortly

**Pull request closed without merging**